docs/rollup-finality-provider-operation.md

```diff
@@ -0,0 +1,798 @@
+> for withdrawing rewards. You **MUST** keep both keys safe.
+
+Optional parameters:
+
+* `--website`: Your finality provider's website
+* `--security-contact`: Contact email for security issues
+* `--details`: Additional description of your finality provider
+* `--daemon-address`: RPC address of the finality provider daemon
+  (default: `127.0.0.1:12581`)
+
+Alternatively, you can create a finality provider by providing a JSON file
+with the finality provider details, similar to the following:
+
+```json
+{
+  "keyName": "The unique key name of the finality provider's Babylon Genesis account",
+  "chainID": "The identifier of the consumer chain",
+  "passphrase": "The pass phrase used to encrypt the keys",
+  "commissionRate": "The commission rate for the finality provider, e.g., 0.05",
+  "moniker": "A human-readable name for the finality provider",
+  "identity": "A optional identity signature",
+  "website": "Validator's (optional) website",
+  "securityContract": "Validator's (optional) security contact email",
+  "details": "Validator's (optional) details",
+  "eotsPK": "The hex string of the finality provider's EOTS public key"
+}
+```
+
+To create a finality provider using the JSON file, you can use the following command:
+
+```shell
+rollup-fpd create-finality-provider --from-file <path-to-json-file>
+```
+
+Upon successful creation, the command will return a JSON response containing
+your finality provider's details:
+
+``` json
+{
+    "finality_provider":
+    {
+      "fp_addr": "bbn1ht2nxa6hlyl89m8xpdde9xsj40n0sxd2f9shsq",
+      "eots_pk_hex":
+      "cf0f03b9ee2d4a0f27240e2d8b8c8ef609e24358b2eb3cfd89ae4e4f472e1a41",
+      "description":
+      {
+        "moniker": "MyFinalityProvider",
+        "website": "https://myfinalityprovider.com",
+        "security_contact": "security@myfinalityprovider.com",
+        "details": "finality provider for the Babylon Genesis network"
+      },
+      "commission": "0.050000000000000000",
+      "status": "REGISTERED"
+    }
+    "tx_hash": "C08377CF289DF0DC5FA462E6409ADCB65A3492C22A112C58EA449F4DC544A3B1"
+}
+Your finality provider is successfully created. Please restart your fpd.
+```
+
+The response includes:
+
+* `fp_addr`: Your Babylon Genesis account address of the finality provider
+* `eots_pk_hex`: Your unique EOTS public key identifier
+* `description`: Your finality provider's metadata
+* `commission`: Your set commission rate
+* `status`: Current status of the finality provider.
+* `tx_hash`: Babylon Genesis transaction hash of the finality provider creation
+  transaction, which you can use to verify the success of the transaction
+  on Babylon Genesis.
+
+### 5.2. Rewards
+
+Rewards are accumulated in a reward gauge, and a finality provider becomes
+eligible for rewards if it has participated sending finality votes.
+The distribution of rewards is based on the provider's voting power portion
+relative to other voters.
```

Comment:
It is actually based on how the rollup operators distributes. We should outline that rewards are happening by rollup operators and link to the babylon genesis reward distribution documentation. We should note that all rewards will be bridged and native x/bank assets.

Reply:
But overall improved https://github.com/babylonlabs-io/finality-provider/pull/573/commits/75addd48c3bd50e90157595ab5fb81715358dfef#diff-69dbcde39256bb776fb499ac39b7aa1642811d6e2dc8294e1d1a7b6b17e370dbR472-R475

Reply:
yep here https://github.com/babylonlabs-io/babylon/blob/release/v3.x/x/btcstaking/docs/rewards-distribution.md

Reply:
Added 2253822bbe7c2df5de74704231e2159c2a7559de